ddtrace/settings/_config.py

```diff
@@ -919,10 +838,8 @@ def _enable_remote_configuration(self):
         from ddtrace.internal.flare.handler import _tracerFlarePubSub
```

Comment:
This might be beyond the scope of this PR but we should also move `AGENT_CONFIG` and `AGENT_TASK` pubsubs out of the global config object.